Wrappers/dinput8/IDirectInputDevice8A.cpp

```diff
@@ -92,28 +92,50 @@ HRESULT m_IDirectInputDevice8A::Acquire()
 {
 	Logging::LogDebug() << __FUNCTION__ << " (" << this << ")";
 
+	LPDIDEVICEINSTANCEA deviceInfo = new DIDEVICEINSTANCEA;
```

Comment:
There is a memory leak here.  You create a new device without ever deleting it later.  Instead this line should be something like this:

```c++
DIDEVICEINSTANCEA deviceInfo;
```

Reply:
Absolutely right, fixed it!